### ISSUE TITLE
Nokogiri cannot be launched.

### ISSUE THREAD

**Issue by diegosempreboni** [alshedivat/al-folio#240]
Dear all, I am trying to launch the bundle, but I get this error. 

bundle exec jekyll serve
bundler: failed to load command: jekyll (/usr/local/bin/jekyll)
Traceback (most recent call last):
	33: from /usr/local/bin/bundle:23:in `<main>'
	32: from /usr/local/bin/bundle:23:in `load'
	31: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/exe/bundle:37:in `<top (required)>'
	30: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/friendly_errors.rb:130:in `with_friendly_errors'
	29: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/exe/bundle:49:in `block in <top (required)>'
	28: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli.rb:24:in `start'
	27: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	26: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli.rb:30:in `dispatch'
	25: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	24: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	23: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	22: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli.rb:494:in `exec'
	21: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:28:in `run'
	20: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:63:in `kernel_load'
	19: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:63:in `load'
	18: from /usr/local/bin/jekyll:23:in `<top (required)>'
	17: from /usr/local/bin/jekyll:23:in `load'
	16: from /Library/Ruby/Gems/2.6.0/gems/jekyll-4.2.0/exe/jekyll:11:in `<top (required)>'
	15: from /Library/Ruby/Gems/2.6.0/gems/jekyll-4.2.0/lib/jekyll/plugin_manager.rb:53:in `require_from_bundler'
	14: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler.rb:173:in `require'
	13: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:50:in `require'
	12: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:50:in `each'
	11: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:61:in `block in require'
	10: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:61:in `each'
	 9: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:66:in `block (2 levels) in require'
	 8: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:66:in `require'
	 7: from /Library/Ruby/Gems/2.6.0/gems/jemoji-0.12.0/lib/jemoji.rb:4:in `<top (required)>'
	 6: from /Library/Ruby/Gems/2.6.0/gems/jemoji-0.12.0/lib/jemoji.rb:4:in `require'
	 5: from /Library/Ruby/Gems/2.6.0/gems/html-pipeline-2.14.0/lib/html/pipeline.rb:3:in `<top (required)>'
	 4: from /Library/Ruby/Gems/2.6.0/gems/html-pipeline-2.14.0/lib/html/pipeline.rb:3:in `require'
	 3: from /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri.rb:11:in `<top (required)>'
	 2: from /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri.rb:11:in `require'
	 1: from /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:6:in `<top (required)>'
/Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:6:in `require': dlopen(/Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle, 0x0009): could not use '/Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle' because it is not a compatible arch - /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle (LoadError)
	34: from /usr/local/bin/bundle:23:in `<main>'
	33: from /usr/local/bin/bundle:23:in `load'
	32: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/exe/bundle:37:in `<top (required)>'
	31: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/friendly_errors.rb:130:in `with_friendly_errors'
	30: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/exe/bundle:49:in `block in <top (required)>'
	29: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli.rb:24:in `start'
	28: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	27: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli.rb:30:in `dispatch'
	26: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	25: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	24: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	23: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli.rb:494:in `exec'
	22: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:28:in `run'
	21: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:63:in `kernel_load'
	20: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:63:in `load'
	19: from /usr/local/bin/jekyll:23:in `<top (required)>'
	18: from /usr/local/bin/jekyll:23:in `load'
	17: from /Library/Ruby/Gems/2.6.0/gems/jekyll-4.2.0/exe/jekyll:11:in `<top (required)>'
	16: from /Library/Ruby/Gems/2.6.0/gems/jekyll-4.2.0/lib/jekyll/plugin_manager.rb:53:in `require_from_bundler'
	15: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler.rb:173:in `require'
	14: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:50:in `require'
	13: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:50:in `each'
	12: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:61:in `block in require'
	11: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:61:in `each'
	10: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:66:in `block (2 levels) in require'
	 9: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.15/lib/bundler/runtime.rb:66:in `require'
	 8: from /Library/Ruby/Gems/2.6.0/gems/jemoji-0.12.0/lib/jemoji.rb:4:in `<top (required)>'
	 7: from /Library/Ruby/Gems/2.6.0/gems/jemoji-0.12.0/lib/jemoji.rb:4:in `require'
	 6: from /Library/Ruby/Gems/2.6.0/gems/html-pipeline-2.14.0/lib/html/pipeline.rb:3:in `<top (required)>'
	 5: from /Library/Ruby/Gems/2.6.0/gems/html-pipeline-2.14.0/lib/html/pipeline.rb:3:in `require'
	 4: from /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri.rb:11:in `<top (required)>'
	 3: from /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri.rb:11:in `require'
	 2: from /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:4:in `<top (required)>'
	 1: from /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:25:in `rescue in <top (required)>'
/Library/Ruby/Gems/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:25:in `require': cannot load such file -- nokogiri/nokogiri (LoadError)

Do you know any issue of al-folio with Big Sur and nokogiri?
How could I try to solve it?

**Comment by Mvrjustid** [comment]
Same here when running
`bundle exec jekyll serve`

got following errors
bundler: failed to load command: jekyll (/Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/bin/jekyll)
Traceback (most recent call last):
	33: from /usr/local/bin/bundle:23:in `<main>'
	32: from /usr/local/bin/bundle:23:in `load'
	31: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/exe/bundle:37:in `<top (required)>'
	30: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/friendly_errors.rb:130:in `with_friendly_errors'
	29: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/exe/bundle:49:in `block in <top (required)>'
	28: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli.rb:24:in `start'
	27: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	26: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli.rb:30:in `dispatch'
	25: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	24: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	23: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	22: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli.rb:494:in `exec'
	21: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:28:in `run'
	20: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:63:in `kernel_load'
	19: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:63:in `load'
	18: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/bin/jekyll:23:in `<top (required)>'
	17: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/bin/jekyll:23:in `load'
	16: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/jekyll-4.2.0/exe/jekyll:11:in `<top (required)>'
	15: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/jekyll-4.2.0/lib/jekyll/plugin_manager.rb:53:in `require_from_bundler'
	14: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler.rb:173:in `require'
	13: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:50:in `require'
	12: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:50:in `each'
	11: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:61:in `block in require'
	10: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:61:in `each'
	 9: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:66:in `block (2 levels) in require'
	 8: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:66:in `require'
	 7: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/jemoji-0.12.0/lib/jemoji.rb:4:in `<top (required)>'
	 6: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/jemoji-0.12.0/lib/jemoji.rb:4:in `require'
	 5: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/html-pipeline-2.14.0/lib/html/pipeline.rb:3:in `<top (required)>'
	 4: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/html-pipeline-2.14.0/lib/html/pipeline.rb:3:in `require'
	 3: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri.rb:11:in `<top (required)>'
	 2: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri.rb:11:in `require'
	 1: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:6:in `<top (required)>'
/Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:6:in `require': 

dlopen(/Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle, 0x0009): could not use '/Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle' because it is not a compatible arch - /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle (LoadError)

	34: from /usr/local/bin/bundle:23:in `<main>'
	33: from /usr/local/bin/bundle:23:in `load'
	32: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/exe/bundle:37:in `<top (required)>'
	31: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/friendly_errors.rb:130:in `with_friendly_errors'
	30: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/exe/bundle:49:in `block in <top (required)>'
	29: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli.rb:24:in `start'
	28: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	27: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli.rb:30:in `dispatch'
	26: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	25: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	24: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	23: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli.rb:494:in `exec'
	22: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:28:in `run'
	21: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:63:in `kernel_load'
	20: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:63:in `load'
	19: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/bin/jekyll:23:in `<top (required)>'
	18: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/bin/jekyll:23:in `load'
	17: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/jekyll-4.2.0/exe/jekyll:11:in `<top (required)>'
	16: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/jekyll-4.2.0/lib/jekyll/plugin_manager.rb:53:in `require_from_bundler'
	15: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler.rb:173:in `require'
	14: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:50:in `require'
	13: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:50:in `each'
	12: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:61:in `block in require'
	11: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:61:in `each'
	10: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:66:in `block (2 levels) in require'
	 9: from /Library/Ruby/Gems/2.6.0/gems/bundler-2.2.16/lib/bundler/runtime.rb:66:in `require'
	 8: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/jemoji-0.12.0/lib/jemoji.rb:4:in `<top (required)>'
	 7: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/jemoji-0.12.0/lib/jemoji.rb:4:in `require'
	 6: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/html-pipeline-2.14.0/lib/html/pipeline.rb:3:in `<top (required)>'
	 5: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/html-pipeline-2.14.0/lib/html/pipeline.rb:3:in `require'
	 4: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri.rb:11:in `<top (required)>'
	 3: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri.rb:11:in `require'
	 2: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:4:in `<top (required)>'
	 1: from /Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:25:in `rescue in <top (required)>'
/Users/joyoung/Public/mvrjustid.github.io/vendor/bundle/ruby/2.6.0/gems/nokogiri-1.11.3-arm64-darwin/lib/nokogiri/extension.rb:25:in `require': 
cannot load such file -- nokogiri/nokogiri (LoadError)

**Comment by alshedivat** [comment]
Thanks for reporting this. Could you try upgrading your ruby version to 2.7.0 or newer and see if the error goes away? Thanks.

**Comment by diegosempreboni** [comment]
I solve it, but not updating the Ruby version as I had the Ruby version 3. However, on my mac, the environment was pointing to the default 2.6 also installed. I had to use echo to change the pointer to the correct ruby version.

**Comment by mukulkhanna** [comment]
I was struggling with the same issue. Apparently, for some reason, the gem being built is that supported by Apple's M1 chip. This worked for me –

```
gem uninstall nokogiri
bundle config set force_ruby_platform true
bundle install
```

StackOverflow source: https://stackoverflow.com/a/67080501

**Comment by jc-nzc** [comment]
> I solve it, but not updating the Ruby version as I had the Ruby version 3. However, on my mac, the environment was pointing to the default 2.6 also installed. I had to use echo to change the pointer to the correct ruby version.

I'm in the same boat. Could you please share your command w/ echo?

In that folder:
`ruby -v`
output
`ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin20]`

But I've upgraded to Ruby 3 🤷🏼‍♂️ 

Thanks!

**Comment by diegosempreboni** [comment]
Try with echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> ~/.zshrc

**Comment by alshedivat** [comment]
To avoid issues with ruby versions, my recommendation is to use [rbenv](https://github.com/rbenv/rbenv).
Closing this issue since it has been resolved, but feel free to continue the discussion here.

**Comment by agmardones** [comment]
> I was struggling with the same issue. Apparently, for some reason, the gem being built is that supported by Apple's M1 chip. This worked for me –
> 
> ```
> gem uninstall nokogiri
> bundle config set force_ruby_platform true
> bundle install
> ```
> 
> StackOverflow source: https://stackoverflow.com/a/67080501

As said [in the docs](https://nokogiri.org/tutorials/installing_nokogiri.html):

> If you are using Bundler
> * version 2.1 or later, then you'll need to run `bundle config set force_ruby_platform true`,
> * version 2.0 or earlier, then you'll need to run `bundle config force_ruby_platform true`

So you must first check your bundler's version. This solved the problem for me (bundler version 1.17.2)
